lib/matplotlib/lines.py

```diff
@@ -568,7 +568,7 @@ def get_window_extent(self, renderer):
                                  ignore=True)
         # correct for marker size, if any
         if self._marker:
-            ms = (self._markersize / 72.0 * self.figure.dpi) * 0.5
```

Comment:
What is this change for?  I assume the issue is that with out this out ticks still get chopped?